### ISSUE TITLE
Finishing the A.I.

### ISSUE THREAD

**Issue by learnitall** [DevelopForLizardz/TicTacTio#3]
The A.I. trainer class needs to be ran in order to produce the neural net for the move making. This then should be implemented into a player class so it can be used in the game (which was the overall goal wasn't it?). However, I don't want the values for the neural network hardcoded, instead there should be some way to load and export neural nets.

**Comment by learnitall** [comment]
Now that the updated code has been pushed, I can use my more powerful machine to generate the A.I. I'll post an estimate on how long it should take before I start. 

**Just a note**: It is probably going to take a long time to get something good, since using neural nets for this kinda problem is a bad idea, therefore I'm going to open a new issue about adding a gradient descent A.I. and an algorithm 'A.I.'

**Comment by learnitall** [comment]
As previously mentioned, here is the new issue: #8

**Comment by learnitall** [comment]
Got around to generating a new A.I., (went for around 220 generations with two pops of 500 neural nets). What was odd was the fitness remained at zero for 200 generations straight and the A.I. will always pick the lower right corner. Ill do some research and mess around with some ideas to see if I can fix it. 

It may just be time to ditch the G.A. technique and replace it with gradient descent.

**Comment by learnitall** [comment]
The problem with the current setup is that the networks are saturated. For now, I'm going to write an algorithm that the A.I. will use, instead of neural networks, so I can 'move on' (even though I haven't added anything to this in a while). I should be able to do this sometime in the next few weeks, as summer begins this week.
